_data/glossary/container-env-variables.yaml

```diff
@@ -0,0 +1,9 @@
+id: container-env-variables
+name: Container Environment Variables
+full-link: /docs/concepts/containers/container-environment-variables.md
+tags:
+- fundamental
+short-description: >
+  Container environment variables are name/value pairs that provide useful information to the container runtime.
```

Comment:
> to the container runtime

to containers, or to container applications.

I'm suggesting this change because "container runtime" usually refers to the engine that creates and manages containers, e.g. 'docker', 'rkt', etc.